packages/compose/src/higher-order/with-is-connected/README.md

```diff
@@ -0,0 +1,19 @@
+# withIsConnected
+
+`withIsConnected` provides a true/false mobile connectivity status based on the [useIsConnected hook](https://github.com/WordPress/gutenberg/blob/aadf8c66ff345b176041ad82e3793191ade5d271/packages/react-native-bridge/index.js#L193-L212).
```

Comment:
Generally speaking, I think it is best to use permalinks, but in the case of documentation residing in this repository I think this would lead to outdated links. 

```suggestion
`withIsConnected` provides a true/false mobile connectivity status based on the `useIsConnected` hook found in the [bridge](https://github.com/WordPress/gutenberg/blob/trunk/packages/react-native-bridge/index.js).
```